geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -2999,6 +2999,7 @@ private boolean putInBucket(final InternalDistributedMember targetNode, final In
 
       try {
         final boolean isLocal = (this.localMaxMemory > 0) && currentTarget.equals(getMyId());
+        logger.info(primaryMoveReadLockAcquired);
```

Comment:
remove this logger statement. I think it broke your build

Reply:
thanks!